src/utils/validation.js

```diff
@@ -0,0 +1,24 @@
+export const Validation = input => {
+	//숫자인지 확인
+	if (isNaN(input)) {
+		throw new Error('[ERROR]')
+	}
+
+	//길이 3글자 확인
+	if (input.trim().length !== 3) {
+		throw new Error('[ERROR]')
+	}
+
+	//0을 포함하는지 확인
+	if (input.includes('0')) {
+		throw new Error('[ERROR]')
+	}
```

Comment:
```suggestion
if (input.includes('0')) {
    throw new Error('[ERROR]')
}
```
0까지 체크하는거 너무 좋네요👍👏🙃 많이 배워갑니다!!

Reply:
우와 저도 이 부분까지는 생각 못했네요..! 👍